test/Conversion/SimToSV/dpi.mlir

```diff
@@ -0,0 +1,94 @@
+// RUN: circt-opt --lower-sim-to-sv %s | FileCheck %s
+// RUN: circt-opt --lower-sim-to-sv --lower-seq-to-sv -export-verilog %s | FileCheck %s --check-prefix=VERILOG
+
+sim.func.dpi @dpi(out arg0: i1, in %arg1: i1, out arg2: i1)
+// CHECK:       sv.func private @dpi(out arg0 : i1, in %arg1 : i1, out arg2 : i1)
+// CHECK-NEXT:  emit.fragment @dpi_dpi_import_fragument {
+// CHECK-NEXT:    sv.func.dpi.import @dpi
+// CHECK-NEXT:  }
+
+// VERILOG:      import "DPI-C" function void dpi( 
+// VERILOG-NEXT:   output arg0,
+// VERILOG-NEXT:   input  arg1,
+// VERILOG-NEXT:   output arg2
+// VERILOG-NEXT: );
```

Comment:
I'm curious what would happen if `%arg1` were an `i32`. Would you get `(output arg0, input int arg1, output arg2)`, or `... output bit arg2)`? I think function arguments in SV inherit the previous argument's type if omitted, similar to types on ports. It might be worthwhile to explicitly emit `bit` for `i1` arguments.